src/style.css

```diff
@@ -1,3 +1,123 @@
 body {
+  --description-and-rate-color: #060b35;
+  --code-and-price-color: #616070;
+  --cardBg-and-button-color: #fff;
+  --cardButton-color: #00acdc;
+  --common-lineHeight: 18px;
+
   margin: 0;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  background-color: #e5e5e5;
+}
+
+h2,
+p {
+  margin: 0;
+}
+
+.card {
+  width: 200px;
+  height: 406px;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: var(--cardBg-and-button-color);
+  box-sizing: border-box;
+}
+
+.card__device-photo {
+  margin: 0 auto 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  margin-bottom: 4px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: var(--common-lineHeight);
+  color: var(--description-and-rate-color);
+}
+
+.card__code {
+  margin-bottom: 16px;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--code-and-price-color);
+}
+
+.card__rateBlock {
+  margin-bottom: 24px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
```

Comment:
add the code from previous task 
![image](https://github.com/mate-academy/layout_product-cards/assets/22824431/b34763c6-8307-4d18-8ff9-cf50b9c41b9e)